src/renderer/views/Watch/Watch.js

```diff
@@ -299,7 +304,17 @@ export default Vue.extend({
           })
 
           this.videoPublished = new Date(result.videoDetails.publishDate.replace('-', '/')).getTime()
-          this.videoDescription = result.player_response.videoDetails.shortDescription
+          try {
+            // workaround for description localization
+            const descriptionLines = result.response.contents.twoColumnWatchNextResults.results.results.contents[1].videoSecondaryInfoRenderer.description.runs
+            this.videoDescription = ''
+            for (let i = 0; i < descriptionLines.length; i++) {
+              this.videoDescription += `${descriptionLines[i].text}\n`
+            }
+          } catch {
```

Comment:
It's a good idea to catch and log the error, that way it can be fixed if it breaks in the future.
```suggestion
          } catch (err) {
            console.error('Failed to extract localised title, falling back to the standard one.', err)
```